Resources/Prototypes/_RMC14/Entities/Structures/Doors/Windoors/windoor.yml

```
@@ -36,6 +36,8 @@
     powerLoad: 0
     needsPower: false
   - type: RMCDropshipBlocked
+  - type: AccessReader
+    containerAccessProvider:
```

Baffles me that YAML interprets this as null but for consistency its better to use null explicitly
```suggestion
    containerAccessProvider: null
```